apps/i18n/common/en_us.json

```diff
@@ -1,5 +1,6 @@
 {
   "and": "and",
+  "animationMode": "Animation",
```

Comment:
Added to the common i18n file because this is where the code mode and design mode strings live.